README.md

```diff
@@ -1,4 +1,4 @@
-### Hi there, I'm Vyom 👋
+<h1 align="center"><em>Hi there, I'm Vyom </em><img src="https://github.com/SparkScratch-P/sparkscratch-p.github.io/blob/main/images/Hi.gif?raw=true" width="50"></h1>
```

Comment:
The image source is not the current repository, add the image to the repository itself rather than using a link, it serves two purposes, first of all, I do not have any fear of losing the link someday, and second somewhat security vulnerability is reduced.